### DISCUSSION TITLE
How to load doctr pytorch weights

### DISCUSSION THREAD

**Discussion by pokecheater** [mindee/doctr#1127] [Q&A]
Hi DocTr-Team/Community,

Inside the tutorial on how to export/import a model everything is only explained for tensorflow:
https://mindee.github.io/doctr/using_doctr/using_model_export.html
<img width="990" alt="image" src="https://user-images.githubusercontent.com/20928617/212320389-6bad4e8f-4b7d-42e7-907d-e7884c5d092f.png">

It is really straightforward. But how should I do this with pytorch?
It is not explained.

Thx in advance

**Comment by felixT2K** [comment]
See https://github.com/mindee/doctr/discussions/940
